<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorsController</class>
 <widget class="QWidget" name="SensorsController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="3">
      <widget class="QComboBox" name="LSDComboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>HeadYaw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="HY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>HeadPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="HP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>AccXvalue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="AccX">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>RShoulderPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="RSP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>AccYvalue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="AccY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="LSP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>LShoulderPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>RShoulderRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="RSR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>AccZvalue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="AccZ">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="LSR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>LShoulderRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>RElbowYaw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="REY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>angX</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="AngX">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="LEY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>LElbowYaw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>RElbowRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="RER">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>angY</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="AngY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QLabel" name="LER">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>LElbowRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>RHipYawPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="RHYP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLabel" name="LHYP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>LHipYawPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>RHipRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="RHR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QLabel" name="LHR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="5">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>LHipRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>RHipPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="RHP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QLabel" name="LHP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="5">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>LHipPitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>RKneePitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="RKP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="4">
      <widget class="QLabel" name="LKP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="5">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>LKneePitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>RAnklePitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="RAP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QLabel" name="LAP">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="5">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>LAnklePitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>RAnkleRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="RAR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="4">
      <widget class="QLabel" name="LAR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="5">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>LAnkleRoll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Support Leg</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="3">
      <widget class="QLabel" name="SupportLeg">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>RFsrRearLeft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="RFsrRL">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QLabel" name="RFsrRR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="3">
      <widget class="QLabel" name="label_52">
       <property name="text">
        <string>RFsrRearRight                          LFsrRearLeft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="17" column="4">
      <widget class="QLabel" name="LFsrRL">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="5">
      <widget class="QLabel" name="LFsrRR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="6">
      <widget class="QLabel" name="label_58">
       <property name="text">
        <string>LFsrRearRight</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>RFsrFrontLeft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="RFsrFL">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QLabel" name="RFsrFR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="3">
      <widget class="QLabel" name="label_53">
       <property name="text">
        <string>RFsrFrontRight                       LFsrFrontLeft</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="18" column="4">
      <widget class="QLabel" name="LFsrFL">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="5">
      <widget class="QLabel" name="LFsrFR">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="18" column="6">
      <widget class="QLabel" name="label_56">
       <property name="text">
        <string>LFsrFrontRight</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
